packages/vuetify/src/components/VCombobox/VCombobox.tsx

```diff
@@ -166,7 +166,7 @@ export const VCombobox = genericComponent<new <
 
     const selections = computed(() => {
       return model.value.map(v => {
-        return items.value.find(item => props.valueComparator(item.value, v.value)) || v
+        return items.value.find(item => props.valueComparator(item.title, v.title)) || v
```

Comment:
How is this related?

Reply:
Not strictly related, could make separate commit.

To see why, go back to `.value` then type `item1` in the combobox in playground, then focus somewhere else. See that it changes `item1` to `Item 1`

Reply:
Hmm yeah I see what you mean, and it only does that because of the transformItem change. I don't know if I really like this change though, it's inconsistent with other components and means if you type `Item 1` then the list item shows as selected even though the model is actually a string instead of the item object (this might be ok though?? idk how people are actually using it)